package.json

```diff
@@ -9,27 +9,26 @@
   },
   "dependencies": {
     "@react-native-community/masked-view": "0.1.10",
-    "@react-navigation/bottom-tabs": "^5.9.1",
-    "@react-navigation/native": "^5.7.5",
-    "@react-navigation/stack": "^5.9.2",
-    "expo": "^39.0.0",
-    "expo-linking": "^1.0.4",
-    "react": "16.13.1",
-    "react-dom": "16.13.1",
-    "react-native": "0.63.3",
-    "react-native-gesture-handler": "~1.7.0",
-    "react-native-reanimated": "~1.13.0",
-    "react-native-safe-area-context": "3.1.4",
-    "react-native-screens": "~2.10.1",
+    "@react-navigation/bottom-tabs": "^6.5.8",
+    "@react-navigation/native": "^6.1.7",
+    "@react-navigation/stack": "^6.3.17",
+    "expo": "^49.0.0",
+    "expo-linking": "~5.0.2",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-native": "0.72.3",
+    "react-native-gesture-handler": "~2.12.0",
+    "react-native-reanimated": "~3.3.0",
+    "react-native-safe-area-context": "4.6.3",
+    "react-native-screens": "~3.22.0",
     "react-native-unimodules": "~0.11.0",
```

Comment:
Just one thing, I think we can remove `"react-native-unimodules": "~0.11.0"` from here?

Reply:
Done